api/overview/azure/dns.md

```diff
@@ -30,10 +33,30 @@ Install-Package Microsoft.Azure.Management.Dns.Fluent
 dotnet add package Microsoft.Azure.Management.Dns.Fluent

```

Comment:
The example below doesn't compile with the `Microsoft.Azure.Management.Dns.Fluent` package. It needs `Microsoft.Azure.Management.Dns`, which is only in preview.